checks/http/roundcube.rb

```diff
@@ -0,0 +1,42 @@
+module Intrigue
+module Ident
+module Check
+class Roundcube < Intrigue::Ident::Check::Base
+
+    def generate_checks(url)
+    [
+        {
+            :type => "fingerprint",
+            :category => "application",
+            :tags => ["Email"],
+            :vendor => "Roundcube",
+            :product => "Roundcube",
+            :website => "https://roundcube.net/",
+            :match_details => "Roundcube title",
+            :version => nil,
+            :match_type => :content_body,
+            :match_content =>  /:: Welcome to Roundcube/i,
+            :paths => ["#{url}"],
+            :inference => false
+        },
+        {
+            :type => "fingerprint",
+            :category => "application",
+            :tags => ["Email"],
```

Comment:
can we add "COTS" as a tag? Is this installed software? COTS lets us know its https://en.wikipedia.org/wiki/Commercial_off-the-shelf  (a good indicator it's probabaly not often updated ;)